src/dynamics/solver/island_solver.rs

```diff
@@ -70,6 +72,14 @@ impl IslandSolver {
                 &self.joint_constraints.position_constraints,
             );
             counters.solver.position_resolution_time.pause();
+        } else {
+            counters.solver.velocity_update_time.resume();
+            bodies.foreach_active_island_body_mut_internal(island_id, |_, rb| {
+                // Since we didn't run the velocity solver we need to integrate the accelerations here
+                rb.integrate_accelerations(params.dt);
```

Comment:
It seems like there is nothing like this `else` in the parallel solver. Is everything part of an island in the parallel case?

Reply:
For both the parallel and serial solvers every non-sleeping body is part of an island. The `else` doesn't exist in the parallel solver because it does not have the `if manifold_indices.len() != 0 || joint_indices.len() != 0 {` either. But I guess it could. This test is just here to avoid entering in code that will just loop on empty arrays.